Resources/public/js/views/services/ez-trashviewservice.js

```diff
@@ -223,6 +223,54 @@ YUI.add('ez-trashviewservice', function (Y) {
         },
 
         /**
+         * Restores trash items
+         *
+         * @method _restoreItems
+         * @protected
+         * @param {Object} e restoreItems event facade
+         * @param {Array} e.trashItems  List of trashItems to be restored
+         */
+        _restoreItems: function (e) {
+            var loadOptions = {api: this.get('capi')},
+                tasks = new Y.Parallel(),
+                service = this,
+                notificationNamePrefix = "restoreTrashItem-";
+
```

Comment:
that's far from being a nitpick, this raises an interesting question on the notifications because one notification per restored content might be a bit too much. Actually, there should be one notification before starting to restore any content which is updated in the `done` callback. And in addition to that, a notification in case of error but with a different id.